Misc/Applications/RNAhybrid/makefile

```diff
@@ -33,6 +33,14 @@ subopt_%:
 		gapc_file="${*}.gap" \
 		gapc_binaryname="$@$(current_windowmodesuffix)";
 
+probing: probing_rnahybrid
+probing_%:
+	$(MAKE) compile \
+		gapc_product="alg_probing * alg_mfe * alg_pretty" \
+		gapc_options="--kbacktrace --no-coopt $(windowmodeflag)" \
```

Comment:
I don't think that suppressing co-optimals here is a good idea. I would remove the `--no-coopt` flag

Reply:
I removed the --no-coopt flag.